code/modules/mob/living/basic/space_fauna/spider/spider_abilities/web.dm

```diff
@@ -83,3 +83,60 @@
 
 /datum/action/cooldown/lay_web/sealer/obstructed_by_other_web()
 	return !!(locate(/obj/structure/spider/stickyweb/sealed) in get_turf(owner))
+
+/datum/action/cooldown/lay_web/solid_web
+	name = "Spin Solid Web"
+	desc = "Spin a web to slow down potential prey."
+	button_icon_state = "lay_solid_web"
+	cooldown_time = 0 SECONDS
+	webbing_time = 5 SECONDS
+
+/datum/action/cooldown/lay_web/solid_web/obstructed_by_other_web()
+	return !!(locate(/obj/structure/spider/solid) in get_turf(owner))
```

Comment:
Is this true? Do you want solid web to be able to be placed on top of non-solid web?
Do you want non-solid webs to be able to be placed on top of solid webs? You are not accounting for either.

Reply:
> Is this true? Do you want solid web to be able to be placed on top of non-solid web?
> 
> Do you want non-solid webs to be able to be placed on top of solid webs? You are not accounting for either.

That's intended for spiders to be able to hide traps and other webs below existing solid webs much like when you put girders and walls above tiles